src/frontend/Komponenter/Personoversikt/Behandlingsoversikt.tsx

```diff
@@ -59,13 +63,37 @@ const Behandlingsoversikt: React.FC<{ fagsakId: string; personIdent: string }> =
         });
     };
 
+    const startRevurdering = (fagsakId: string) => {
+        axiosRequest<Ressurs<void>, { fagsakId: string }>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/revurdering/${fagsakId}`,
+            data: { fagsakId: fagsakId },
+        }).then((response) => {
+            if (response.status === RessursStatus.SUKSESS) {
+                hentFagsak();
+            }
+        });
+    };
+
     useEffect(() => {
         if (fagsakId) {
             hentFagsak();
         }
         // eslint-disable-next-line
     }, [fagsakId]);
 
+    useEffect(() => {
+        if (fagsak.status === RessursStatus.SUKSESS) {
+            const alleBehandlingerErFerdige =
+                fagsak.data.behandlinger.length > 0 &&
+                fagsak.data.behandlinger.find(
+                    (behandling) => behandling.status !== BehandlingStatus.FERDIGSTILT
+                ) === undefined;
+            settKanStarteRevurdering(alleBehandlingerErFerdige);
```

Comment:
Sjekkes også backend, men dette burde holde for ikke vise knapp når det ikke er mulig å revurdere.